calico_containers/calico_ctl/profile.py

```diff
@@ -130,7 +133,7 @@ def profile(arguments):
     elif arguments.get("add"):
         profile_add(arguments.get("<PROFILE>"))
     elif arguments.get("remove"):
-        profile_remove(arguments.get("<PROFILE>"))
+        profile_remove(arguments.get("<PROFILE>"), arguments.get("--no-check"))
```

Comment:
since nocheck is actually a keyword argument, it is generally better to be explicit by using a keyword argument in the invocation.

Alternatively, since we are always passing in nocheck, make it a positional arg instead and drop the keyword.